apps/mobile/src/libs/components/checkbox-categories-form/libs/helpers/get-categories-sorted-by-score/get-categories-sorted-by-score.helper.ts

```diff
@@ -0,0 +1,11 @@
+import { type QuizScoresGetAllItemResponseDto } from "~/libs/types/types";
+
+const getCategoriesSortedByScore = (
```

Comment:
Is it sorted ASC or DESC?  Be more specific with the name. Or add a second argument, which will receive order type